src/pages/resultsView/mutation/mutations.scss

```diff
@@ -3,3 +3,11 @@
         margin-right: 5px;
     }
 }
+
```

Comment:
both of these are too general because they could appear elsewhere and we wouldn't want this behavior.   the best thing to do would be to convert this into a css module.